circle.yml

```diff
@@ -0,0 +1,31 @@
+machine:
+  java:
+    version: openjdk8
+  environment:
+    SBT_VERSION: 0.13.15
+dependencies:
+  pre:
+    - wget -q https://dl.bintray.com/sbt/debian/sbt-$SBT_VERSION.deb
+    - sudo dpkg -i sbt-$SBT_VERSION.deb
+    - ./ci/before-install.sh
+  cache_directories:
+    - "~/.ivy2"
+    - "~/.m2"
+    - "~/.sbt"
+test:
+  override:
+    - case $CIRCLE_NODE_INDEX in 0) sbt ++2.11.8 test ;; 1) sbt ++2.11.11 test ;; 2) sbt ++2.12.2 test ;; esac:
+        parallel: true
+  post:
+    - mkdir -p $CIRCLE_TEST_REPORTS/junit/
+    - find . -type f -regex ".*/target/test-reports/.*xml" -exec cp {} $CIRCLE_TEST_REPORTS/junit/ \;
```

Comment:
They show up in the test summary on CircleCI. If you have a failure, the test appears here, and in the email.

<img width="1102" alt="screen shot 2017-05-12 at 9 20 50 am" src="https://cloud.githubusercontent.com/assets/19369/25999766/616feff8-36f4-11e7-808f-f20baee2ca24.png">